PROJECT

```diff
@@ -71,4 +69,31 @@ resources:
   kind: Runner
   path: github.com/mercedes-benz/garm-operator/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: mercedes-benz.com
+  group: garm-operator
+  kind: GarmServerConfig
+  path: github.com/mercedes-benz/garm-operator/api/v1alpha1
+  version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: mercedes-benz.com
+  group: garm-operator
+  kind: Endpoint
```

Comment:
would rename to GithubEndpoint here.
Kubernetes itself has the concept of endpoints - just do a kubectl get endpoint and you will get endpoints behind a service

Reply:
See [686f43a](https://github.com/mercedes-benz/garm-operator/pull/158/commits/686f43a9bbaddc29acec86c686f167b20226e4ae)